zmconfigd/smtpd_sender_restrictions.cf

```diff
@@ -2,7 +2,7 @@
 %%contains VAR:zimbraMtaSmtpdSenderRestrictions check_sender_access lmdb:/opt/zextras/conf/postfix_reject_sender%%
 %%contains VAR:zimbraServiceEnabled cbpolicyd^ check_policy_service inet:localhost:%%zimbraCBPolicydBindPort%%%%
 %%contains VAR:zimbraServiceEnabled amavis^ check_sender_access regexp:/opt/zextras/common/conf/tag_as_originating.re%%
-permit_mynetworks
+permit_mynetworks, reject_sender_login_mismatch
```

Comment:
hello, after the introduction of this parameter we are not able to send mail from outlook or any other tool.
We are discussing in your forum https://community.zextras.com/forum/carbonio-general-thread/connection-problems-smtps-ports-465-and-587/ and now I understood why this behavior.
For now I modified manually the config file but I think you need to rollback it or adjust other things.